src/ast.scm

```diff
@@ -226,13 +226,13 @@
                               ""))
                         "")
                     (string.rep "    " ilvl) "end"))
-	   ((do)
-	    (let ((call (cadr e))
-		  (args (cdr (cadr (caddr e))))
-		  (body (caddr (caddr e))))
-	      (deparse-block (string (deparse call) " do" (if (null? args) "" " ")
-				     (deparse-arglist args))
-			     (cdr body) ilvl)))
+       ((do)
```

Comment:
It looks like this block moved to the wrong indent (I am not sure it was right before either, as seems it was not aligned with the (try) above but was aligned with the (struct) below it)

Reply:
Looks right to me